packages-rs/drainpipe/src/schema.rs

```diff
@@ -4,7 +4,9 @@ diesel::table! {
     dead_letter_queue (rowid) {
         rowid -> Integer,
         seq -> BigInt,
-        msg -> Text,
+        err_kind -> Integer,
```

Comment:
this file is autogenerated.  should we be worried that it specifies this column as an `Integer` rather than a `SmallInt`?

Reply:
running the `diesel setup` and other commands regenerated this differently, so I updated the rest of the code to use an `Integer` instead of `SmallInt`.